gcampus/core/receivers.py

```diff
@@ -117,6 +117,18 @@ def update_measurement_indices(sender, instance: Parameter, **kwargs):
         instance.measurement.structure_index.update()
 
 
+@receiver(post_save, sender=Parameter)
+def check_warning_limits(sender, instance: Parameter, **kwargs):
+    if instance.parameter_type.lower_warning_limit is not None:
```

Comment:
You can return after setting `requires_review = True` for the first time. The second case does not have to be checked and given that `upper_warning_limit` > `lower_warning_limit` it should never be true anyway.